packages/crowdstrike/data_stream/fdr/elasticsearch/ingest_pipeline/default.yml

```
@@ -49,7 +49,11 @@ processors:
         - UNIX_MS
         - ISO8601
       ignore_failure: true
-      if: ctx.event?.created == null
+      if: >
+        ctx.event?.created == null &&
+        ctx.crowdstrike?.UTCTimestamp != null &&
+        ctx.crowdstrike?.UTCTimestamp != "" &&
+        ctx.crowdstrike?.UTCTimestamp != "none"
```

```suggestion
        ctx.crowdstrike?.UTCTimestamp != null &&
        ctx.crowdstrike.UTCTimestamp != "" &&
        ctx.crowdstrike.UTCTimestamp != "none"
```
For the rest of the cases as well.